### DISCUSSION TITLE
Spinner doesn't spin without holding key on "Autopilot" mod

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#16801] [Q&A]
entire of this [replay](https://cdn.rukchadisa.live/r/Whistle_Python_-_Laur_-_Pristine_[Viola]_(2565-01-28)_Osu-1.osr)  
spinner wont spin until i hold my key even though i am on auto pilot mod
logs: [logs](https://cdn.rukchadisa.live/r/logs_2.zip)

**Comment by frenzibyte** [comment]
I've reworded the title to simplify the purpose of this discussion, which is making osu!lazer autospin spinners instead of having to manually hold the key during it, to match behaviour with osu!stable.
